public/index.php

```diff
@@ -19,7 +19,7 @@
 // Instantiate PHP-DI ContainerBuilder
 $containerBuilder = new ContainerBuilder();
 
-if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'production') {
+if (isset($_ENV['PHP_ENV']) && $_ENV['PHP_ENV'] === 'prod') {
```

Comment:
@okiedork should this be kept as "production"?

Reply:
`dev`.

Reply:
Ok just kidding, `prod` here.